frontend/playwright-tests/covid_cases_deaths_hospitalizations.nightly.spec.ts

```diff
@@ -0,0 +1,114 @@
+import { test } from '@playwright/test'
+
+test('Covid Cases, Deaths, Hospitalizations', async ({ page }) => {
```

Comment:
Let's actually split it into 3 test() functions inside this one single file. Then it'll be easier to identify a test failure in a particular datatype. Just have each new test start right before where you're having it go to local host (and make those relative urls)